pkg/serverutils/utils.go

```diff
@@ -2,19 +2,16 @@ package serverutils
 
 import (
 	"encoding/json"
-	"github.com/coreos/pkg/capnslog"
 	"net/http"
-)
 
-var (
-	plog = capnslog.NewPackageLogger("github.com/openshift/console", "serverutils")
+	"k8s.io/klog"
 )
 
 // Copied from Server package to maintain error response consistency
 func SendResponse(rw http.ResponseWriter, code int, resp interface{}) {
 	enc, err := json.Marshal(resp)
 	if err != nil {
-		plog.Printf("Failed JSON-encoding HTTP response: %v", err)
+		klog.V(4).Infof("Failed JSON-encoding HTTP response: %v", err)
```

Comment:
...since it's an error
```suggestion
		klog.Errorf("Failed JSON-encoding HTTP response: %v", err)
```